ui/components/ui/icon/copy-contract-details.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
This icon can be used for any copy instances so we can update the file and component name to be more generic 
1. Update file name from `copy-contract-details.js` to `icon-copy.js`
2. Update component name from `CopyContractDetails` to `IconCopy`

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).